src/main/java/gov/cms/madie/terminology/service/FhirTerminologyService.java

```diff
@@ -0,0 +1,41 @@
+package gov.cms.madie.terminology.service;
+
+import ca.uhn.fhir.context.FhirContext;
+import ca.uhn.fhir.parser.IParser;
+import gov.cms.madie.models.measure.ManifestExpansion;
+import gov.cms.madie.terminology.models.UmlsUser;
+import gov.cms.madie.terminology.webclient.FhirTerminologyServiceWebClient;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.hl7.fhir.r4.model.Bundle;
+import org.springframework.cache.annotation.Cacheable;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+@Slf4j
+@RequiredArgsConstructor
+public class FhirTerminologyService {
+  FhirContext ctx = FhirContext.forR4();
+  private final FhirTerminologyServiceWebClient fhirTerminologyServiceWebClient;
+
+  @Cacheable("manifest-list")
```

Comment:
do we need to have time to live for cache or is there a default ttl by spring/boot? I don't think it changes often bu just thinking

Reply:
Yeah I am not sure either, but to force refresh the list ( which is expected not to change often ) I thought we could just restart our service